#### Standard Deviation : 
S.D is a number that describes how spread out the values are.
OR : The square root of the variance; 
###### A low standard deviation means that most of the numbers are close to the mean (average) value.mA high standard deviation means that the values are spread out over a wider range.

In [14]:
import numpy as np 

In [ ]:
#Using numpy std()
speed = [61,62,63,64,61,72,67,68]
x = np.std(speed)
x


np.float64(3.665719574653795)

In [ ]:
# Similarly if we Take a Large Range of number like 
speed = [32,111,138,28,59,77,97]
x = np.std(speed)
x

np.float64(37.84501153334721)

#### Variance :
Variance is another number that shows how spread out the data/values are.
If we multiply the S.D by itself, we get the variance ; i.e S.D^2



##### Manually Calculation :
To Calculate Variance we have to Follow Four Steps :
1. Find mean
2. For each value, find the difference from the mean
3. For each difference: find the square value
4. The variance is the average number of these squared differences:

In [ ]:
#1. Mean :
(61 + 62 + 63 + 64 + 61 + 72 + 67 + 68 ) / 8


64.75

In [5]:
#2. For each value, the Difference from the Mean
print (61 - 64.75)
print(62 - 64.75)
print(63 - 64.75)
print (64 - 64.75 )
print (61 - 64.75)
print (72 - 64.75)
print(67 - 64.75)
print(68 - 64.75)


-3.75
-2.75
-1.75
-0.75
-3.75
7.25
2.25
3.25


In [10]:
#3. For Each Difference, taking Square
print((-3.75)**2)
print((-2.75)**2)
print((-1.75)**2)
print((-0.75)**2)
print((-3.75)**2)
print(7.25**2)
print(2.25**2)
print(3.25**2)


14.0625
7.5625
3.0625
0.5625
14.0625
52.5625
5.0625
10.5625


In [11]:
(14.0625 + 7.5625 + 3.0625 + 0.5625 + 14.0625 + 52.5625 + 5.0625 + 10.5625) /8 

13.4375

therefore, 13.4375 is the Variance 
###### From this value we can find the  S.D if we take the Square Root, it will be;  3.665719574653795

In [18]:
#Using numpy  var() 
speed = [61,62,63,64,61,72,67,68]
x = np.var(speed)
x

np.float64(13.4375)

In [17]:
y = np.std(speed)
y

np.float64(3.665719574653795)